cmd/admin-kms-key-status.go

```diff
@@ -86,6 +86,6 @@ func (s kmsKeyStatusMsg) String() string {
 }
 
 func (s kmsKeyStatusMsg) JSON() string {
-	const fmtStr = `{"key-id":"%s","encryption-error":"%s","update-error":"%s","decryption-error":"%s"}`
-	return fmt.Sprintf(fmtStr, s.KeyID, s.EncryptionErr, s.UpdateErr, s.DecryptionErr)
+	const fmtStr = `{"key-id":"%s","encryption-error":"%s","decryption-error":"%s"}`
```

Comment:
Why is this change here in this PR?

Reply:
Please remove this change

Reply:
Compilation will fail without this change. I can remove it once a changeset is merged that has removed instances of UpdateErr in mc. UpdateErr has been removed in minio codebase, which is what is causing the compilation error in mc.

Reply:
Still don't understand the need for this change. If this were the case it should fail even now without any of your other changes, as I do not see this PR using the latest minio package in go.mod

Reply:
You should also update minio package in go.mod, then this change will make sense